frontend/svelte/src/lib/i18n/en.json

```
@@ -272,7 +272,8 @@
     "confirm_adopt_headline": "Adopt Proposal",
     "confirm_adopt_text": "You are about to cast $votingPower votes for this proposal, are you sure you want to proceed?",
     "confirm_reject_headline": "Reject Proposal",
-    "confirm_reject_text": "You are about to cast $votingPower votes against this proposal, are you sure you want to proceed?"
+    "confirm_reject_text": "You are about to cast $votingPower votes against this proposal, are you sure you want to proceed?",
+    "vote_status": "$neuronId has $vote"
```

Just tested it with a screen reader, sounds already great. What do you think about this tiny improvement?
```suggestion
    "vote_status": "Neuron $neuronId has voted $vote"
```